end-to-end-test/remote/specs/core/groupComparison.screenshot.spec.js

```diff
@@ -206,120 +206,186 @@ describe('group comparison page screenshot tests', function() {
             assertScreenShotMatch(res);
         });
 
-        it('group comparison page mutation enrichments tab several groups', function() {
-            browser.click('.tabAnchor_mutations');
-            browser.waitForVisible(
-                'div[data-test="GroupComparisonMutationEnrichments"]',
-                10000
-            );
-            browser.moveToObject('body', 0, 0);
-            var res = browser.checkElement('.msk-tab:not(.hiddenByPosition)', {
-                hide: ['.qtip'],
-            });
-            assertScreenShotMatch(res);
-        });
-
-        it('group comparison page mutation enrichments tab patient mode', function() {
-            browser.execute(function() {
-                groupComparisonStore.setUsePatientLevelEnrichments(true);
-            });
-            browser.waitForVisible(
-                'div[data-test="GroupComparisonMutationEnrichments"]',
-                10000
-            );
-            browser.moveToObject('body', 0, 0);
-            var res = browser.checkElement('.msk-tab:not(.hiddenByPosition)', {
-                hide: ['.qtip'],
-            });
-            assertScreenShotMatch(res);
-        });
-
-        it('group comparison page mutation enrichments tab 20 genes with highest frequency in any group', function() {
-            browser.execute(function() {
-                groupComparisonStore.setUsePatientLevelEnrichments(false);
-            });
-            browser.waitForVisible(
-                'div[data-test="GroupComparisonMutationEnrichments"]',
-                10000
-            );
-            browser.click('[data-test="selectGenes"]');
-            var input = $('input[data-test=numberOfGenes]');
-            input.setValue('20\n');
-            browser.waitForEnabled('[data-test="addGenestoBarPlot"]', 10000);
-            browser.click('[data-test="addGenestoBarPlot"]');
-            browser.waitForVisible('div[data-test="GeneBarPlotDiv"]', 10000);
-            browser.moveToObject('body', 0, 0);
-            var res = browser.checkElement('div[data-test="GeneBarPlotDiv"]', {
-                hide: ['.qtip'],
-            });
-            assertScreenShotMatch(res);
-        });
-
-        it('group comparison page mutation enrichments tab gene box highest average frequency', function() {
-            browser.click('[data-test="selectGenes"]');
-            browser.execute(function() {
-                genesSelection.onGeneListOptionChange({
-                    label: 'Genes with highest average frequency',
+        it.omit(
```

Comment:
Well, since these are remote e2e-tests these depend on live-portal API. As such these tests can be created/reactivated when the whole feature is accepted. Or is there another way to make these tests against a different portal?